### ISSUE TITLE
REQUEST_INSTALL_PACKAGES permission

### ISSUE THREAD

**Issue by binwiederhier** [binwiederhier/ntfy#684]
I received this question via email:

> I was reading about your software ntfy, and I think it's great !
But I was shocked to know that it requires the "REQUEST_INSTALL_PACKAGES" permission.  Why ?
> I also noticed that the GooglePlay version doesn't need this permission, but the F-Droid version declares it.
Why the difference between the two versions?

**Comment by binwiederhier** [comment]
There is a bit more background on it here: https://github.com/binwiederhier/ntfy/issues/531

But the summary is: When you send an .apk file to ntfy as an attachment, Android will not allow you to install the .apk unless the application that downloaded the file has the REQUEST_INSTALL_PACKAGES permission. This applies to trying to install the .apk from ntfy itself, as well as when you try to install it from a file browser. It was wild.

This at least was the case on my device. But when discussing this in Discord it was not everyone's experience. 

Honestly I am happy to remove the permission. I don't think many people will miss it.

**Comment by Quantum-IA** [comment]
I agree with Google's view explained in #531 (https://github.com/binwiederhier/ntfy/issues/531): "Permission use is not directly related to your app's core purpose".
Why sending an .APK as a ntfy attachment?  To make updates?
I don't feel comfortable knowing that an .APK could be installed automatically from within ntfy.
I prefer having this permission removed.
Thank you.

**Comment by Quantum-Future** [comment]
Hello, the current version (1.16.0) was released on Dec 11, 2022.
Do you have plans for releasing an updated version without this permission and some bugfixes or new features?

**Comment by Quantum-Future** [comment]
> Honestly I am happy to remove the permission. I don't think many people will miss it.

Hello Phil, please remember this for the next Android release.
Thank you !

**Comment by binwiederhier** [comment]
Done https://github.com/binwiederhier/ntfy-android/commit/876d01da00fce68e889eaa6e10270eeef0b0b1da

**Comment by Quantum-Future** [comment]
Hi, the latest version added to F-Droid on Sept 23 (v1.17.8), still has this permission.

**Comment by Quantum-Future** [comment]
Ver 1.17.13 was published today on F-Droid and still declares this permission.
